core/src/banking_stage/qos_service.rs

```diff
@@ -198,10 +198,19 @@ impl QosService {
                 // Only transactions that the qos service included have to be
                 // checked for update
                 if let Ok(tx_cost) = tx_cost {
-                    if let CommitTransactionDetails::Committed { compute_units } =
-                        transaction_committed_details
+                    if let CommitTransactionDetails::Committed {
+                        compute_units,
+                        loaded_accounts_data_size,
+                    } = transaction_committed_details
                     {
-                        cost_tracker.update_execution_cost(tx_cost, *compute_units)
+                        cost_tracker.update_execution_cost(
+                            tx_cost,
```

Comment:
I'm fine if we want to do this outside of this PR, but it would probably be more clear if this was named `estimated_tx_cost`, `tx_cost_estimated`, or `tx_cost_budgeted`